js/private/js_binary.bzl

```diff
@@ -129,6 +129,10 @@ _ATTRS = {
         and ["Make variable"](https://bazel.build/reference/be/make-variables) substitution if `expand_env` is set to True.
         """,
     ),
+    "env_inherit": attr.string_list(
```

Comment:
I think this should actually only be set for the `_test` version, while `env` seems to apply to both `*_binary` and `*_test` rules (see https://bazel.build/reference/be/common-definitions#common-attributes-binaries).

Can we add it down below in the `js_test = rule(...)` statement? Or if you want you can add a `_TEST_ATTRS = ` that merges `_ATTRS` plus a new set of additional `js_test` attributes.